nss-cohort-37/Workforce-Red-Jaguars
# Jv employee details

# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Look at details of employee. Make sure field names look correct in all view models.
# Checklist:
